command/flag/arguments.go

```diff
@@ -205,7 +205,6 @@ type RenameServiceArgs struct {
 
 type ShareServiceArgs struct {
```

Comment:
I guess the entire struct is not needed anymore - yet some other commands use this pattern anyways so up to you